editor/plugins/gdextension/cpp_scons/template/addon/library_name.gdextension

```diff
@@ -0,0 +1,28 @@
+[configuration]
+
+entry_symbol = "__LIBRARY_NAME___library_init"
+compatibility_minimum = __GODOT_VERSION__
+reloadable = true
+
+[libraries]
+
+macos.debug = "bin/lib__LIBRARY_NAME__.macos.debug.framework"
```

Comment:
There have recently been some changes to the [godot-cpp-template version](https://github.com/godotengine/godot-cpp-template/blob/main/demo/bin/example.gdextension) that I think should be applied here too. In particular, relative paths require a leading `./`, and the binary itself should be linked instead of the `.framework`, because of an unfixed bug https://github.com/godotengine/godot/issues/96403.

(this is one of the reasons i'm trying to get rid of `.framework` in the first place, because then we could just use `.dylib` and have less trouble with this weird bug)